app/anti_virus_check.py

```diff
@@ -138,9 +138,8 @@ def _get_anti_virus_result(self, data_id):
 
         self._check_av_response(response)
 
-        self.bound_logger.info("Response from AV", response=response.text)
-
         result = response.json()
+        self.bound_logger.info("Response from AV", response=result)
```

Comment:
Do we really need to be logging the full JSON response if the file has passed AV and/or if the scan has not yet completed?

Reply:
Good point, probably not. I'll change.